src/views/GameView.vue

```diff
@@ -6,8 +6,11 @@
     </template>
     <!-- Authenticated View -->
     <template v-else>
-      <div id="game-menu-wrapper" class="d-flex flex-column align-center">
-        <game-menu />
+      <div id="game-menu-wrapper" class="d-flex flex-column justify-center align-center">
+        <div class="d-flex align-center justify-center btnWrapper">
```

Comment:
I think this would achieve the same column affect ([docs](https://vuetifyjs.com/en/styles/flex/)). 
```suggestion
        <div class="d-flex align-center justify-center flex-column">
```